### PR TITLE
Devi/device io controller

### PR DESCRIPTION
**Pull request by v-dmudili**
Created a folder for Device IO Control Uwp app & added all the project files for windows 10 Iot Enterprise.
Following changes are done:

1) file modified : DeviceIoControlUwp.vcxproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.
    TargetPlatform Toolset v141 to v142.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the app *packages installing steps

### CODE REVIEW

**Review by rsameser — APPROVED** [review]

File: DeviceIOContoller/CPP/Package.appxmanifest [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft. All rights reserved. -->
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="16684029-29b1-4d3c-a32e-1da372cc3136"
```
Comment (neilsh-msft) [inline DeviceIOContoller/CPP/Package.appxmanifest:10]: Name="DeviceIOControlUWP"

File: DeviceIOContoller/CPP/README.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+```
+
+The output will be a long string of comma-separated numbers that we will paste into a registry file. Create a file named deviceac.reg with the following contents:
+
+    Windows Registry Editor Version 5.00
+
+    [HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001\Enum\ACPI\PNP0501\1]
+    "Security"=hex:1,0,4,90,0,0,0,0,0,0,0,0,0,0,0,0,14,0,0,0,2,0,60,0,4,0,0,0,0,0,14,0,0,0,0,10,1,1,0,0,0,0,0,5,12,0,0,0,0,0,18,0,0,0,0,10,1,2,0,0,0,0,0,5,20,0,0,0,20,2,0,0,0,0,14,0,0,0,0,10,1,1,0,0,0,0,0,5,B,0,0,0,0,0,18,0,0,0,0,10,1,2,0,0,0,0,0,F,2,0,0,0,1,0,0,0
+
+Copy this file to c:\data on your MBM and run:
```
Comment (neilsh-msft) [inline DeviceIOContoller/CPP/README.md]: This is referencing Minnow Board Max (MBM).  It needs to be updated for UP Board.

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

File: DeviceIOContoller/CPP/Package.appxmanifest (hunk 1 above)
Comment (v-abdrahman) [reply DeviceIOContoller/CPP/Package.appxmanifest:10]: Updated

File: DeviceIOContoller/CPP/README.md (hunk 2 above)
Comment (v-abdrahman) [reply DeviceIOContoller/CPP/README.md]: Content for MBM is removed

**Pull request merged**